modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -131,3 +131,43 @@ $ oc adm node-logs ci-ln-m0wpfjb-f76d1-vnb5x-master-0 --path=oauth-apiserver/aud
 ----
 {"kind":"Event","apiVersion":"audit.k8s.io/v1","level":"Metadata","auditID":"dd4c44e2-3ea1-4830-9ab7-c91a5f1388d6","stage":"ResponseComplete","requestURI":"/apis/user.openshift.io/v1/users/~","verb":"get","user":{"username":"system:serviceaccount:openshift-monitoring:prometheus-k8s","groups":["system:serviceaccounts","system:serviceaccounts:openshift-monitoring","system:authenticated"]},"sourceIPs":["10.0.32.4","10.128.0.1"],"userAgent":"dockerregistry/v0.0.0 (linux/amd64) kubernetes/$Format","objectRef":{"resource":"users","name":"~","apiGroup":"user.openshift.io","apiVersion":"v1"},"responseStatus":{"metadata":{},"code":200},"requestReceivedTimestamp":"2021-03-08T17:47:43.653187Z","stageTimestamp":"2021-03-08T17:47:43.660187Z","annotations":{"authorization.k8s.io/decision":"allow","authorization.k8s.io/reason":"RBAC: allowed by ClusterRoleBinding \"basic-users\" of ClusterRole \"basic-user\" to Group \"system:authenticated\""}}
 ----
+
+* View the OpenShift OAuth server audit logs:
+
+.. List the OpenShift OAuth server audit logs that are available for each control plane node:
++
+[source,terminal]
+----
+$ oc adm node-logs --role=master --path=oauth-server/
+----
++
+.Example output
+[source,terminal]
+----
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit-2022-05-11T18-57-32.395.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-0 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit-2022-05-11T19-07-07.021.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-1 audit.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit-2022-05-11T19-06-51.844.log
+ci-ln-m0wpfjb-f76d1-vnb5x-master-2 audit.log
+----
+
+.. View a specific OpenShift OAuth server log by providing the node name and the log name:
```

Comment:
> View a specific OpenShift OAuth server log
View a specific OpenShift OAuth server audit log

Same for other places "View a specific OpenShift API server log", "View a specific Kubernetes API server log", "View a specific OpenShift OAuth API server log"